packages/@aws-cdk/aws-batch/lib/compute-environment.ts

```diff
@@ -57,6 +73,16 @@ export interface ComputeResources {
    */
   readonly instanceRole?: iam.IRole;
 
+  /**
+   * An optional launch template to associate with your compute resources. To use a launch template, you must
+   * specify either the launch template ID or launch template name in the request, but not both.
+   * For more information, see Launch Template Support.
```

Comment:
`see Launch Template Support`? Did you mean to link to the README content you added?

Reply:
My bad, some heavy copy pasting on my side. I'll fix.